pages/Teams/team-profile-page.cfm

```diff
@@ -70,21 +70,31 @@
 </cfquery>
 
 <cfquery name="getLeadingScorer" datasource="roundleague">
-  SELECT ps.PlayerID, ps.points, s.SeasonName, t.teamName, s.seasonID, p.firstName, p.lastName
```

Comment:
So the correct expression for this is called CTE - Common Table Expression. Different then temp table I mentioned earlier. Either way cool use of GPT resources lol